### PR TITLE
Replace instances.Instance proto with v1.Instance proto

### PR DESCRIPTION
**Pull request by dhurley**
### Proposed changes

Replace instances.Instance proto with v1.Instance proto

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [ ] I have read the [`CONTRIBUTING`](https://github.com/nginx/agent/blob/main/docs/CONTRIBUTING.md) document
- [ ] I have run ```make install-tools``` and have attached any dependency changes to this pull request
- [ ] If applicable, I have added tests that prove my fix is effective or that my feature works
- [ ] If applicable, I have checked that any relevant tests pass after adding my changes
- [ ] If applicable, I have updated any relevant documentation ([`README.md`](https://github.com/nginx/agent/blob/main/README.md))
- [ ] If applicable, I have tested my cross-platform changes on Ubuntu 22, Redhat 8, SUSE 15 and FreeBSD 13

### CODE REVIEW

**Review by oliveromahony — APPROVED** [review]

File: api/grpc/mpi/v1/command.proto [hunk 1]
```diff
@@ -124,6 +138,26 @@ message AgentConfig {
...
+    string config_path = 3;
+}
+
+// A set of runtime NGINX configuration that gets populated 
+message NGINXPlusConfig {
+    // master process id
+    int32 process_id = 1;
+    // where the binary location is, if empty, this is a remote instance
+    string binary_path = 2;
+    // where the configguration files are located
```
Comment (Dean-Coakley) [inline api/grpc/mpi/v1/command.proto:157]:
```suggestion
    // where the configuration files are located
```

File: api/grpc/mpi/v1/command.proto [hunk 2]
```diff
@@ -124,6 +138,26 @@ message AgentConfig {
     string message_buffer_size = 6;
 }
 
+// A set of runtime NGINX configuration that gets populated 
+message NGINXConfig {
+    // master process id
+    int32 process_id = 1;
+    // where the binary location is, if empty, this is a remote instance
+    string binary_path = 2;
+    // where the configguration files are located
```
Comment (Dean-Coakley) [inline api/grpc/mpi/v1/command.proto:147]:
```suggestion
    // where the configuration files are located
```

**Review by Dean-Coakley — APPROVED** [review]

File: api/grpc/mpi/v1/command.proto (hunk 1 above)
Comment (dhurley) [reply api/grpc/mpi/v1/command.proto]: updated

File: api/grpc/mpi/v1/command.proto (hunk 2 above)
Comment (dhurley) [reply api/grpc/mpi/v1/command.proto]: updated

**Pull request merged**